src/config/config.cc

```diff
@@ -204,6 +204,9 @@ Config::Config() {
       {"rocksdb.write_options.low_pri", true, new YesNoField(&RocksDB.write_options.low_pri, false)},
       {"rocksdb.write_options.memtable_insert_hint_per_batch", true,
        new YesNoField(&RocksDB.write_options.memtable_insert_hint_per_batch, false)},
+
+      /* rocksdb read options */
+      {"rocksdb.read_options.async_io", false, new YesNoField(&RocksDB.read_options.async_io, true)},
```

Comment:
Should it be YesNoField(&RocksDB.read_options.async_io, **false**) here?

Reply:
yes, I missed this